pyproject.toml

```diff
@@ -110,6 +110,8 @@ dev = [
     "nbmake",
     # To access the metadata for python packages
     "importlib-metadata; python_version < '3.10'",
+    # To fetch data from pybamm-data
+    "pooch",
```

Comment:
Could you make this a required dependency? Also, could you update the installation guide in the documentation about this – we have a table there that lists all of our dependencies and their minimum versions. It might also be good to add a minimum version here (one that is available for Python 3.9+ on both conda-forge and PyPI).